modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -74,487 +71,440 @@ const ProcessorInfo ParallelCoordinates::getProcessorInfo() const { return proce
-    tf_.set(tf);
-    tf_.setCurrentStateAsDefault();
-
-    tf.clear();
-    tf.add(0.5, vec4(1, 0, 0, 1));
-    tfSelection_.set(tf);
-    tfSelection_.setCurrentStateAsDefault();
-
-    handle_ = util::make_unique<SimpleMesh>(DrawType::Triangles, ConnectivityType::Strip);
-    auto handleMesh = dynamic_cast<SimpleMesh *>(handle_.get());
-    handleMesh->addVertex(vec3(0, 0, 0), vec3(0, 0, 0), vec4(0, 0, 0, 0));
-    handleMesh->addVertex(vec3(0, 1, 0), vec3(0, 1, 0), vec4(0, 0, 0, 0));
-    handleMesh->addVertex(vec3(1, 0, 0), vec3(1, 0, 0), vec4(0, 0, 0, 0));
-    handleMesh->addVertex(vec3(1, 1, 0), vec3(1, 1, 0), vec4(0, 0, 0, 0));
-    handleMesh->addIndices(0, 1, 2, 3);
-
-    axis_ = util::make_unique<SimpleMesh>(DrawType::Lines, ConnectivityType::None);
-    auto axisMesh = dynamic_cast<SimpleMesh *>(axis_.get());
-    axisMesh->addVertex(vec3(0, 0, 0), vec3(0, 0, 0), vec4(1, 0, 0, 1));
-    axisMesh->addVertex(vec3(0, 1, 0), vec3(0, 1, 0), vec4(0, 1, 0, 1));
-    axisMesh->addIndices(0, 1);
-
-    std::vector<int> fontSizes = {8, 10, 11, 12, 14, 16, 20, 24, 28, 36, 48, 60, 72, 96};
-    for (auto size : fontSizes) {
-        std::string str = toString(size);
-        fontSize_.addOption(str, str, size);
-        valuesFontSize_.addOption(str, str, size);
+        lineShader_.onReload([&]() { this->invalidate(InvalidationLevel::InvalidOutput); });
+        lineShader_.build();
     }
-    fontSize_.setSelectedIndex(6);
-    fontSize_.setCurrentStateAsDefault();
-    valuesFontSize_.setSelectedIndex(4);
-    valuesFontSize_.setCurrentStateAsDefault();
-
-    fontSize_.onChange([this]() { textCacheDirty_ = true; });
-    valuesFontSize_.onChange([this]() { textCacheDirty_ = true; });
-
-    auto app = InviwoApplication::getPtr();
-    auto module = app->getModuleByType<PlottingGLModule>();
-    auto path = module->getPath(ModulePath::Images);
-    std::string filename = path + "/pcp_handle.png";
-    auto factory = app->getDataReaderFactory();
-    if (auto reader = factory->getReaderForTypeAndExtension<Layer>("png")) {
-        auto outLayer = reader->readData(filename);
-        auto ram = outLayer->getRepresentation<LayerRAM>();
-        outLayer->setDataFormat(ram->getDataFormat());
-
-        handleImg_ = std::make_shared<Image>(outLayer);
-        handleImg_->getRepresentation<ImageRAM>();
-    } else {
-        LogError("Failed to read handle image");
-    }
-
-    lineShader_.onReload([&]() { this->invalidate(InvalidationLevel::InvalidOutput); });
-    axisShader_.onReload([&]() { this->invalidate(InvalidationLevel::InvalidOutput); });
-    handleShader_.onReload([&]() { this->invalidate(InvalidationLevel::InvalidOutput); });
 
-    dataFrame_.onChange([&]() {
-        createOrUpdateProperties();
-        recreateLines_ = true;
-        textCacheDirty_ = true;
-    });
-    selectedColorAxis_.onChange([&]() { recreateLines_ = true; });
+    dataFrame_.onChange([&]() { createOrUpdateProperties(); });
```

Comment:
This will change properties during network evaluation.

Reply:
yes